test/correctly/import_nested/proto/greet/v1/greet.proto

```
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package greet;
+
+import "greet/v1/common/common.proto";
```

Buf (1.55.1)

import "greet/v1/common/common.proto": file does not exist

(COMPILE)